index.html

```diff
@@ -7,12 +7,71 @@
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
+
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
 
+    <!-- Header-->
+    <header>
+        <img class="icon" src="img/karma-logo.svg" alt="logo of Karma" />
+    
+    <!-- Nav Bar-->
+        <nav class="navbar">
+            <ul>
+                <li><a class="active" href="#sec-img-background">Meet Karma</a></li>
+                <li><a href="#sec-main-body">How It Work</a></li>
+                <li><a href="#sec-main-body">Store</a></li>
+                <li><a href="#footer">log</a></li>
+                <li><a href="#footer">Help</a></li>
+                <li><a href="#footer">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <!-- Main Hero-->
+    <main>
+        <section>
+           <div class="container">
+            <p class="intro-karma">Introducing Karma</p>
+            <p class="intro-bring">Bring WiFi with you, everywhere you go.</p>
+            <input type="button" value="Learn More" />
+        </div>
+        </section>
+        <p class="little-karma">Everyone needs a little karma.</p>
+        <section class="main-body" id="main-body">
+           
+            <div>
+                <img class="ibody" src=" img/icon-devices.svg" alt="Internet"/>
+                <p>Internet for all devices</p>
+            </div>
+
+            <div>
+                <img class="ibody" src="img/icon-coffee.svg" alt="Boost"/>
+                <p>Boost your productivity</p>
+            </div>
+
+            <div>
+                <img class="ibody" src="img/icon-refill.svg" alt="Pay"/>
+                <p> Pay as You Go</p>
+            </div> </section>
```

Comment:
Watch out for your formatting and indententation. `</section>` should be on the next line.